lib/pbio/test/src/test_color_light.c

```diff
@@ -45,6 +46,15 @@ static const pbio_color_light_funcs_t test_light_funcs = {
     .set_hsv = test_light_set_hsv,
 };
 
+/**
+ * Handles pending events to ensure animation has started and first frame
+ * has been activated.
+ */
+static void wait_for_first_animation_frame(void) {
```

Comment:
This function would be better name something like `handle_pending_events` since it isn't actually waiting for any specific condition other than pending events.

And since other tests need this too, might as well make this a helper function accessible to all tests (e.g. a static inline function in `test-pbio.h`).